java/compaction-core/src/main/java/sleeper/compaction/job/CompactionFactory.java

```diff
@@ -37,19 +38,28 @@ public class CompactionFactory {
     private static final Logger LOGGER = LoggerFactory.getLogger(CompactionFactory.class);
 
     private final String tableName;
-    private final String tableBucket;
-    private final String fs;
+    private final String outputFilePrefix;
     private final String iteratorClassName;
     private final String iteratorConfig;
 
     public CompactionFactory(InstanceProperties instanceProperties, TableProperties tableProperties) {
```

Comment:
Would this class be better called CompactionJobFactory?

Reply:
I've renamed it.